litmus-portal/frontend/src/views/CreateWorkflow/ReliabilityScore/index.tsx

```diff
@@ -52,22 +54,25 @@ const ReliablityScore = () => {
         <div className={classes.mainDiv}>
           <div>
             <Typography className={classes.headerText}>
-              <strong>
-                Adjust the weights of the experiments in the workflow
-              </strong>
+              <strong>{t('createWorkflow.reliabilityScore.header')}</strong>
             </Typography>
             <Typography className={classes.description}>
-              You have selected {weights?.length} tests in the “Kubernetes
-              conformance test” workflow. Successful outcome of each test
-              carries a certain weight. We have pre-selected weights for each
-              test for you. However, you may review and modify the weigtage
-              against. <strong>The weights are relative to each other.</strong>
+              {t('createWorkflow.reliabilityScore.info')} {weights?.length}{' '}
+              <Trans i18nKey="createWorkflow.reliabilityScore.infoNext">
+                tests in the “Kubernetes conformance test” workflow. Successful
+                outcome of each test carries a certain weight. We have
+                pre-selected weights for each test for you. However, you may
+                review and modify the weigtage against.
+                <strong>The weights are relative to each other.</strong>
+              </Trans>
```

Comment:
Trans might not be the right fit if we are not interpolating any dynamic translation data